core/src/test/java/org/pentaho/platform/engine/security/SecurityHelperTest.java

```diff
@@ -40,10 +42,7 @@
 import org.junit.Test;
 import org.mockito.AdditionalMatchers;
 import org.mockito.Matchers;
-import org.pentaho.platform.api.engine.IPentahoObjectFactory;
-import org.pentaho.platform.api.engine.IPentahoSession;
-import org.pentaho.platform.api.engine.ISystemSettings;
-import org.pentaho.platform.api.engine.IUserRoleListService;
+import org.pentaho.platform.api.engine.*;
```

Comment:
I guess IntelliJ was not your friend here and _"optimized"_ the imports.
Can you please revert this? Thanks!

Reply:
@pentaho-lmartins sure. Pull request updated